local-cli/runIOS/runIOS.js

```diff
@@ -82,9 +82,11 @@ function _runIOS(argv, config, resolve, reject) {
     '-derivedDataPath', 'build',
   ];
   console.log(`Building using "xcodebuild ${xcodebuildArgs.join(' ')}"`);
-  child_process.spawnSync('xcodebuild', xcodebuildArgs, {stdio: 'inherit'});
+  const xcodeBuildProcess = child_process.spawnSync('xcodebuild', xcodebuildArgs, {stdio: 'pipe'});
+  console.log(xcodeBuildProcess.output[1].toString());
 
-  const appPath = `build/Build/Products/Debug-iphonesimulator/${inferredSchemeName}.app`;
+  const productName = xcodeBuildProcess.output[1].toString().match(/Touch (build\/Build\/Products\/.*\/.*\.app)/);
```

Comment:
Could we instead parse some of the `xcodebuildArgs` and get the scheme that way?